guide/14-deep-learning/how_AutoDL_works.ipynb

```diff
@@ -0,0 +1,566 @@
+{
```

Comment:
<p><code>This method will return a dataframe</code> to </p><p><code>This method will return a evaluation report as a dataframe that will include several fields including model's accuracy </code>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1198/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>